skimage/feature/_canny.py

```diff
@@ -11,11 +11,13 @@
 All rights reserved.
 Original author: Lee Kamentsky
 """
+from __future__ import division, print_function, absolute_import
```

Comment:
But do you mean that we should have this line in every file in skimage, even when we don't use the function for example? (I understood what this line was doing, but not why you were adding it).